File: Code-Forces/734A.cpp
#include<bits/stdc++.h>

using namespace std;
int main() {
    long n,an=0,da=0;
    string s;
    cin>>n>>s;
    for (int i=0;i<n;i++){
        if (s[i]=='A'){an++;}
        else{da++;}
    }
    if (an>da) cout<<"Anton"<<endl;
    else if (da>an) cout <<"Danik" <<endl;
    else if (da==an) cout<<"Friendship";
    
    return 0;
}

File: Code-Forces/734A.java


import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        long n = input.nextLong();
        long an = 0;
        long da = 0;
        String s = input.next();
        char[] sc = s.toCharArray();
        for (int i = 0; i < n; i++) {
            if (sc[i] == 'A') {
                an++;
            } else {
                da++;
            }
        }
        if (an > da)
            System.out.println("Anton");
        else if (da > an)
            System.out.println("Danik");
        else if (da == an)
            System.out.println("Friendship");
    }
}

